### PR TITLE
Update logging message on MultipleJGitEnvironmentRepository, JGitEnvironmentRepository

### PR DESCRIPTION
**Pull request by ndy2**
As decribe in #2161, it is more easy to debug by throw original exception.
It can be handled by adding original exception as suppressed to master branch exception in case both findOne try thorws exception.

Thanks for @woshikid to sharing solutions.

### CODE REVIEW

**Comment by woshikid** [comment]
in `JGitEnvironmentRepository#getLocations` there is another Try Master Branch try catch. maybe we should do the same thing here.

**Comment by ndy2** [comment]
I will check and add !

**Comment by woshikid** [comment]
I debugged and found the actually Try Master Branch behavior happens in `JGitEnvironmentRepository#getLocations` in normal situation.

**Comment by ndy2** [comment]
I found that there are actually three case where
`tryMasterBranch` flag in `JGitEnvironmentRepository` is used as flag for retry some logic

1.` JGitEnvironmentRepository#getLocations`
2. `JGitEnvironmentRepository#checkoutDefaultBranchWithRetry`
3.` MultipleJGitEnvironmentRepository#findOne`

two methods in JGitEnvironmentRepository handle original exception by logging.

Considering that, i think simple loggin on `MultipleJGitEnvironmentRepository#findOne` would be consistent solution for this issue.

So I add a log on `MultipleJGitEnvironmentRepository#findOne` refer to `JGitEnvironmentRepository`.

And fix a wrong log message in `JGitEnvironmentRepository`

File: spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/MultipleJGitEnvironmentRepository.java [hunk 1]
```diff
@@ -160,7 +160,6 @@ public Environment findOne(String application, String profile, String label, boo
 							this.logger.debug("Cannot load configuration from " + candidate.getUri() + ", cause: ("
 									+ e.getClass().getSimpleName() + ") " + e.getMessage(), e);
 						}
-						continue;
```
Comment (ryanjbaxter) [inline spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/MultipleJGitEnvironmentRepository.java:163]: Why is this being removed?

**Review by ryanjbaxter — CHANGES_REQUESTED** [review]

File: spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/MultipleJGitEnvironmentRepository.java (hunk 1 above)
Comment (ndy2) [reply spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/MultipleJGitEnvironmentRepository.java:163]: My IDE suggests
```
'continue' is unnecessary as the last statement in a loop 
```

so i removed it.
I find that there is one more such continue statement at `MultipleJGitEnvironmentRepository.getLocations`

May i remove 'continue;' from that also or retreive this one.

Comment (spencergibb) [reply spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/MultipleJGitEnvironmentRepository.java:163]: please limit changes to those specific to the issue.

**Comment by ndy2** [comment]
After I restore the continue statement. 
BootstrapConfigServerIntegrationTests.contextLoads fails...

I don't know why it fails. Since the change i made is only to add some logging.

**Comment by ryanjbaxter** [comment]
@ndy2 can you please pull the latest changes from 3.1.x into your branch?

**Pull request merged**